musicplatform/albums/admin.py

```diff
@@ -1,13 +1,36 @@
-from socket import fromshare
-from django.contrib import admin
-from .models import Album
+from logging import exception
 from django import forms
+from django.contrib import admin, messages
+from .models import Album, Song
+
+class SongInlineFormSet(forms.models.BaseInlineFormSet):    
```

Comment:
@yousefokasha12 
Hi Yousef, 
Thanks for the review, but about the (create an album -> add some songs -> delete them -> save the album) situation, I handle it here, and I just tested it and it works as expected. 
Could you please test it again, and if it still not working, could you give me some details to remake the problem. Thanks.